enhancements/microshift/dummy-gateway-interface-ovn.md

```diff
@@ -0,0 +1,426 @@
+---
+title: dummy-gateway-interface-ovn
+authors:
+- "@pliurh"
+reviewers:
+- "@zshi-redhat, MicroShift"
```

Comment:
We should include @pmtk and @pacevedom here, too.

Should we include anyone else from the networking teams, at least for awareness?